src/frontend/src/pages/WorkPackageForm/CreateWorkPackageForm.tsx

```diff
@@ -20,7 +21,7 @@ const CreateWorkPackageForm: React.FC = () => {
   return (
     <WorkPackageForm
       wbsNum={validateWBS(wbsNum)}
-      mutateAsync={mutateAsync}
+      mutateAsync={mutateAsync as UseMutateAsyncFunction<unknown, unknown, unknown>}
```

Comment:
can we get rid of this? and just have it say mutateAsync = {mutateAsync}?

Reply:
If I leave it as mutateAsync={mutateAsync}, I receive a problem and fail the typescript check

Reply:
Can you try "as MutationFunction" instead?